tests/system/feature_store/test_spark_engine.py

```diff
@@ -721,3 +729,49 @@ def test_error_is_properly_propagated(self):
                 spark_context=self.spark_service,
                 run_config=fs.RunConfig(local=False),
             )
+
+    @pytest.mark.parametrize("engine", ["spark"])
+    def test_get_offline_features_with_filter(self, engine):
+        key = "patient_id"
+        measurements = fs.FeatureSet(
+            "measurements",
+            entities=[fs.Entity(key)],
+            timestamp_key="timestamp",
+            engine="spark",
```

Comment:
You actually ignore the `engine` parameter 😄 
Just get rid of it.

Reply:
Thanks, I forgot to delete it.